lib/omniscrapper_hq/operations/schedule/index.rb

```diff
@@ -0,0 +1,32 @@
+module Operations
+  module Schedule
+    class Index < Base
+      GRAPHQL_CLIENT ||= System::Container['omniscrapper.graphql_client']
+
+      Query ||= GRAPHQL_CLIENT.parse <<-'GRAPHQL'
+        query {
+          events {
+            id,
+            name,
+            taskId,
+            frequencyPeriod,
+            frequencyQuantity
+          }
+        }
+      GRAPHQL
+
+      def call
+        results = GRAPHQL_CLIENT.query(Query)
+        results.data.events.map do |event|
+          ::Event.new(
```

Comment:
It should be called `Schedule`, not `Event`.
We keep the period when task is performed.